appsettings/appsettings_test.go

```diff
@@ -29,17 +30,18 @@ const appSettingsUnsetFixture string = `
 {
     "owner": "test",
     "app": "unset-test",
-	  "maintenance": true,
+	"maintenance": true,
```

Comment:
Apparently so - that was the case before so I am sure it "duplicate" when i added extra lines. I can remove real quick if that is supposed to be spaces?